src/components/NetworkLink.tsx

```diff
@@ -0,0 +1,40 @@
+import type { FC } from "react";
+import { Button, Icon } from "@canonical/react-components";
+import { useNavigate } from "react-router-dom";
+
+interface Props {
+  address: string;
+  instanceName: string;
+  projectName: string;
+}
+
+export const NetworkLink: FC<Props> = ({
```

Comment:
I am not 100% sure about the layout. Maybe the pencil icon should be left aligned before the address? Or maybe not pushed to the right-hand side of the column, but appear right next to the IP if the column fits the whole IP? Can you reach out to design to get an informed opinion on this? Personally not sure, might be also be good as is.

Another thought: Maybe moving the IP address column all the way to the right will solve the design challenge here. Because usually we have the actions on the right-hand side of the row and having the IP as last column would be aligning with the other pages.


<img width="1651" height="957" alt="image" src="https://github.com/user-attachments/assets/126b02cc-ccd9-4bb2-a3d4-cb73dd68ed62" />